README.md

```diff
@@ -194,3 +194,30 @@ For small deployments (e.g a kubernetes test cluster/AIO K8s deployment) you can
 ```console
 ~$ make image
 ```
+
+# Limitations
+## Ethernet
+For Mellanox Hardware, due to kernel limitation, it is required to pre-allocate MACs for all VFs in the deployment
+in the following manner:
+
+__Set VF administrative MAC address :__
+
+```
+$ ip link set <pf-netdev> vf <vf-index> mac <mac-address>
+```
+
+__Unbind/Bind VF driver :__
+
+```
+$ echo <vf-pci-address> > /sys/bus/pci/drivers/mlx5_core/unbind
+$ echo <vf-pci-address> > /sys/bus/pci/drivers/mlx5_core/bind
+```
+
+__Example:__
+```
+$ ip link set enp4s0f0 vf 3 mac 02:03:00:00:48:56
+$ echo 0000:03:00.5 > /sys/bus/pci/drivers/mlx5_core/unbind
+$ echo 0000:03:00.5 > /sys/bus/pci/drivers/mlx5_core/bind
+```
+In addition, MAC address must not be changed via CNI plugins unless they utilize the flow described
```

Comment:
Thinking of this, im not sure this is needed, unless there is some validation by Firmware/Driver for the encapsulated RoCE packet.

Reply:
yes let wait for Mamduh result and see